src/main/java/com/yahoo/sketches/counting/FrequentItems.java

```diff
@@ -0,0 +1,107 @@
+package com.yahoo.sketches.counting;
+
+import java.util.ArrayList;
```

Comment:
I could give a long lecture on the inefficiencies of the Java generic collections classes, especially for production pipeline applications that need to be small and fast. Your comment: "clocks at ~2M updates per second";  My sketches can update at 100M per second and for long streams they approach 200M/sec.  
But I'll let it go for now  :)